src/admin/sql/install.mysql.utf8.sql

```diff
@@ -7,6 +7,7 @@ CREATE TABLE IF NOT EXISTS `#__tj_activities` (
 	`target` text(900),
 	`target_id` text(900),
 	`type` text(900),
+	`client` text(900),
```

Comment:
data type for client should be varchar and not text

Reply:
Done